src/Jira/Api/Client/CurlClient.php

```diff
@@ -75,6 +76,10 @@ public function sendRequest(
 
 		if ( $method == 'GET' ) {
 			$url .= '?' . http_build_query($data);
+
+			if ( !is_array($data) ) {
```

Comment:
This needs to be moved up. Otherwise you already get notice about wrong parameter usage in `http_build_query` call.